lib/OrganizationsFieldArray/OrganizationsFieldArray.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { uniq } from 'lodash';
+import { Field } from 'redux-form';
+
+import {
+  Button,
+  Col,
+  IconButton,
+  Layout,
+  Row,
+  Select,
+} from '@folio/stripes/components';
+
+import withKiwtFieldArray from '../withKiwtFieldArray';
+import { required } from '../validators';
+import OrganizationPicker from './OrganizationPicker';
+
+class OrganizationsFieldArray extends React.Component {
+  static propTypes = {
+    addContactBtnLabel: PropTypes.node,
+    change: PropTypes.func,
+    isEmptyMessage: PropTypes.node,
+    items: PropTypes.arrayOf(PropTypes.object),
+    name: PropTypes.string.isRequired,
+    onAddField: PropTypes.func.isRequired,
+    onDeleteField: PropTypes.func.isRequired,
+    roles: PropTypes.arrayOf(PropTypes.object),
+    organizationPickerComponent: PropTypes.elementType,
+  };
+
+  static defaultProps = {
+    addContactBtnLabel: <FormattedMessage id="stripes-erm-components.organizations.addOrganization" />,
+    isEmptyMessage: <FormattedMessage id="stripes-erm-components.organizations.noOrganizations" />,
+    organizationPickerComponent: OrganizationPicker,
+  }
+
+  state = {
+    organizations: {},
+  };
+
+  static getDerivedStateFromProps(props, state) {
```

Comment:
Do you need to be merging everything into `state` for this component? I don't really see `state` being used anywhere, though I may be missing something. 

(Internal contacts does it mainly because we need to merge the previously-set contacts with complete user info that's fetched from the `/users` endpoint.)

Reply:
Good catch. We don't _state_ at all in this file